interfaces/cython/cantera/yamlwriter.pyx

```diff
@@ -9,6 +9,10 @@ cdef class YamlWriter:
         self._writer.reset(new CxxYamlWriter())
         self.writer = self._writer.get()
 
+    def add_header(self, _SolutionBase soln):
+        """ Include top-level information for the specified Solution object """
+        self.writer.setHeader(soln.base.header())
```

Comment:
I suppose the Python function name ought to be updated to match the C++ function name here.